templates/emails/contact.html.twig

```diff
@@ -0,0 +1,8 @@
+<ul>
+    <li>Sujet : {{ message.subject }}</li>
```

Comment:
What's the point of repeating the subject and the mail, they already are in in the mail headers !